Sites/GD2/Project 1/Styles.css

```
@@ -0,0 +1,50 @@
+body {
+    margin: 100px;
+}
+
+h1 {
+    font-family: 'Noto Sans TC', sans-serif;
+    color: #260648;
+    font-weight: bold;
+    font-size: 45px;
+}
+
+h2 {
+    font-family: 'Noto Sans TC', sans-serif;
+    color: #6B4B8C;
+    font-weight: normal;
+    font-size: 30px;
+    text-align: left;
+}
+
+div {
+    margin: 40px;
+}
+
+blockquote {
+    font-family: 'PT Sans', sans-serif;
+    font-weight: italic;
```

`font-weight` controls the thickness of the font while `font-style` controls if it is italic or not:

```suggestion
    font-style: italic;
```